lib/OpenQA/CacheService/Plugin/Helpers.pm

```diff
@@ -5,27 +5,19 @@ package OpenQA::CacheService::Plugin::Helpers;
 use Mojo::Base 'Mojolicious::Plugin';
 use Time::Seconds;
 
-sub register {
-    my ($self, $app) = @_;
+sub register ($self, $app) {
 
     # To determine download progress and guard against parallel downloads of the same file
     $app->helper('progress.downloading_job' => \&_progress_downloading_job);
     $app->helper('progress.is_downloading' => \&_progress_is_downloading);
     $app->helper('progress.guard' => \&_progress_guard);
 }
 
-sub _progress_downloading_job {
-    my ($c, $lock) = @_;
-    return $c->downloads->find($lock);
-}
+sub _progress_downloading_job ($c, $lock) { return $c->downloads->find($lock) }
```

Comment:
The `return` is not needed, perl will always return the result of the last statement.
Same below